editor/sidecar/aot.clj

```diff
@@ -22,7 +22,8 @@
 
 (def excluded-sources-by-build-type
   {"release" ["src/clj/dev.clj"
-              "src/clj/prof.clj"]})
+              "src/clj/prof.clj"
+              "src/clj/yourkit.clj"]})
```

Comment:
Hmm, I don't like that we keep dev-time code in the source for production code and then exclude it where necessary. I think a simpler approach would be to keep it separately, e.g. in `src/yourkit`, and then include it on the classpath during the development. This is done e.g. for Reveal with `:reveal` profile in `project.clj`.

Reply:
Makes sense. I'll move these files to `src/dev` and add it as a `source-path` in the `:dev` profile.